### PR TITLE
Add notistack for notification handling.

### PR DESCRIPTION
**Pull request by waldenraines**
## PR Type

* [ ] Bug fix
* [x] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactor (i.e., no functional changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation changes
* [ ] Other (please describe below):

## Description

Use notistack for notifications and add a notification when copying a `<Copyable>`.  Also remove unused `<ErrorNotification>` component.

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

### CODE REVIEW

**Comment by waldenraines** [comment]
Peek 2020-04-21 18-16

**Comment by bucko13** [comment]
Nice! Thank you. What do you think of putting a bit of a preview string of what was copied? Like "2n5j... copied to clipboard"? Not sure if it's even the right thing to do but wondering if it could be helpful

**Comment by waldenraines** [comment]
> Nice! Thank you. What do you think of putting a bit of a preview string of what was copied? Like "2n5j... copied to clipboard"? Not sure if it's even the right thing to do but wondering if it could be helpful

Good idea, I can give that a shot.

**Comment by waldenraines** [comment]
> Good idea, I can give that a shot.

It's a little strange on longer values and I don't see a way to set a maximum length:

gscreenshot_2020-04-21-193901

Another thing you can do is set `preventDuplicates` so only one of the "Copied to clipboard" message is displayed at a time.

**Comment by waldenraines** [comment]
@bucko13 updated

File: src/components/Copyable.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import React, { useState, useEffect } from "react";
 import CopyToClipboard from "react-copy-to-clipboard";
 import { FileCopy } from "@material-ui/icons";
 import { IconButton, makeStyles } from "@material-ui/core";
+import { useSnackbar } from "notistack";
```
Comment (bucko13) [inline src/components/Copyable.jsx:8]: Wow, cool. Hooks for the snackbar!

Comment (bucko13) [reply src/components/Copyable.jsx:8]: Or at least that's what it looks like it's doing...

**Comment by bucko13** [comment]
Are we sure the error notification wasn't being used? It seemed like it was connected to the redux store which could have an error message in it to be displayed by the notification snack bar.

File: src/components/Copyable.jsx (hunk 1 above)
Comment (waldenraines) [reply src/components/Copyable.jsx:8]: Yep!

**Comment by waldenraines** [comment]
> Are we sure the error notification wasn't being used? It seemed like it was connected to the redux store which could have an error message in it to be displayed by the notification snack bar.

I've never seen one personally (and I've caused a lot of errors :grin:) but I'm fine restoring that file just in case.  It seems like maybe it used to be used but isn't any longer?  Regardless I can restore it just for safety's sake.

**Comment by waldenraines** [comment]
> Regardless I can restore it just for safety's sake.

@bucko13 updated.

**Comment by bucko13** [comment]
Thank you! My guess is that we're probably not handling the errors properly. Should maybe be using this instead of console logging.

**Comment by bucko13** [comment]
Confirmed that it is hooked up and theoretically being used in the test runner and the KeystorePicker. It can be used with the errorNotificationAction dispatcher.

**Pull request merged**